requirements.txt

```diff
@@ -1,9 +1,10 @@
 pytest>=3.5.0
 pytest-black
-numpy
+numpy==1.20
```

Comment:
If you are using isclose do you have to hardcode the numpy version?

Reply:
Good catch

Reply:
BTW, fixing numpy==1.20 was not good enough to produce the result giving the same number of significant digits. Finding out what exact package was causing this was going to be a treasure hunt promising little prize..